src/block/Sugarcane.php

```diff
@@ -49,6 +49,15 @@ public function getStateBitmask() : int{
 		return 0b1111;
 	}
 
+	private function seekToBottom() : Sugarcane{
```

Comment:
Instead of forcing this to be a `Sugarcane` instance, it would be better to return a `Vector3` or `Position`, and allow `grow()` to accept it as a parameter.